confmap/provider.go

```diff
@@ -33,7 +33,7 @@ import (
 //		// repeat retrieve/wait/close cycle until it is time to shut down the Collector process.
 //		// ...
 //		mapProvider.Shutdown()
```

Comment:
Should the example here be updated to remove references to `mapProvider`?

Reply:
Done.